app/scripts/notice-controller.js

```diff
@@ -58,6 +58,23 @@ module.exports = class NoticeController extends EventEmitter {
     }
   }
 
+  markAllNoticesRead (cb) {
```

Comment:
if possible, lets not introduce any more callback-based methods. lets use async/await